tests/openwisp2/asgi.py

```diff
@@ -0,0 +1,6 @@
+import os
+
+from django.core.asgi import get_asgi_application
+
+os.environ.setdefault("DJANGO_SETTINGS_MODULE", "openwisp2.settings")
```

Comment:
Why do we need this here? I don't see it in openwisp-controller and openwisp-notifications.